app/jobs/dradis/plugins/projects/project_clone_job.rb

```diff
@@ -0,0 +1,47 @@
+module Dradis::Plugins::Projects
```

Comment:
I don't think this class should live in the plugin but instead it should be in Pro. The main reason is that this is a Pro-only feature and it wouldn't make sense for it to be available in CE.

Reply:
Got it, will move to the main app.